travis-ci/diffs/0001-bpf-selftests-temporary-remove-tests-impacted-by-LLV.diff

```diff
@@ -0,0 +1,889 @@
+From 4d20285988055fbd1eadb683ca47280660fc6a5e Mon Sep 17 00:00:00 2001
+From: Mykola Lysenko <mykolal@fb.com>
+Date: Thu, 14 Apr 2022 12:05:37 -0700
+Subject: [PATCH bpf-next] bpf/selftests: temporary remove tests impacted by
+ LLVM crash
+
+Signed-off-by: Mykola Lysenko <mykolal@fb.com>
+---
+ tools/testing/selftests/bpf/Makefile          |   4 +-
+ .../selftests/bpf/prog_tests/test_profiler.c  |  72 ---
+ tools/testing/selftests/bpf/prog_tests/usdt.c | 421 ------------------
+ tools/testing/selftests/bpf/progs/profiler1.c |   6 -
+ tools/testing/selftests/bpf/progs/profiler2.c |   6 -
+ tools/testing/selftests/bpf/progs/profiler3.c |   6 -
+ .../bpf/progs/test_core_reloc_enumval.c       |  72 ---
+ .../selftests/bpf/progs/test_urandom_usdt.c   |  70 ---
+ tools/testing/selftests/bpf/progs/test_usdt.c |  96 ----
+ .../selftests/bpf/progs/test_usdt_multispec.c |  32 --
+ 10 files changed, 2 insertions(+), 783 deletions(-)
+ delete mode 100644 tools/testing/selftests/bpf/prog_tests/test_profiler.c
+ delete mode 100644 tools/testing/selftests/bpf/prog_tests/usdt.c
+ delete mode 100644 tools/testing/selftests/bpf/progs/profiler1.c
+ delete mode 100644 tools/testing/selftests/bpf/progs/profiler2.c
+ delete mode 100644 tools/testing/selftests/bpf/progs/profiler3.c
+ delete mode 100644 tools/testing/selftests/bpf/progs/test_core_reloc_enumval.c
+ delete mode 100644 tools/testing/selftests/bpf/progs/test_urandom_usdt.c
+ delete mode 100644 tools/testing/selftests/bpf/progs/test_usdt.c
+ delete mode 100644 tools/testing/selftests/bpf/progs/test_usdt_multispec.c
+
+diff --git a/tools/testing/selftests/bpf/Makefile b/tools/testing/selftests/bpf/Makefile
+index bafdc5373a13..f9a8b251e0a0 100644
+--- a/tools/testing/selftests/bpf/Makefile
++++ b/tools/testing/selftests/bpf/Makefile
+@@ -334,7 +334,7 @@ SKEL_BLACKLIST := btf__% test_pinning_invalid.c test_sk_assign.c
+ 
+ LINKED_SKELS := test_static_linked.skel.h linked_funcs.skel.h		\
+ 		linked_vars.skel.h linked_maps.skel.h 			\
+-		test_subskeleton.skel.h test_subskeleton_lib.skel.h	\
++		test_subskeleton.skel.h test_subskeleton_lib.skel.h	#\
+ 		test_usdt.skel.h
```

Comment:
probably want to comment this out?

Reply:
my understanding this should work https://stackoverflow.com/questions/4493291/how-to-add-multi-line-comments-in-makefiles

Reply:
oh, wrapping the # with this \... fancy, ok :)